src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -380,6 +384,10 @@ public Study updateStudyCustodians(User user, Integer studyId, String custodians
     } else {
```

Comment:
It's not part of this PR, but the above code fragment
```
   Optional x = stream.filter(something).findFirst();
   if (x.isPresent) { ...
```
can be replaced with
```
  boolean b = stream.anyMatch(something);
  if (b) { ...
```
as long as `x` isn't needed, which is true in this case.

You can also replace the ternary op with `Objects.requireNonNullElse()` as long as you qualify the `Set.of()` type:
```
  Objects.requireNonNullElse(study.getProperties(), Set.<StudyProperty>of()).stream()
```